tests/test_browser_model.py

```diff
@@ -11,17 +10,22 @@ class TestBrowserModel(object):
     def test_new_private_window(self, selenium):
         """Tests opening a new private browsing window via menu"""
         foxpuppet = FoxPuppet(selenium)
+        wm = foxpuppet.window_manager
```

Comment:
I would either call `foxpuppet.window_manager` each time, or name this variable `window_manager`. Naming it `wm` makes it hard to read.